### DISCUSSION TITLE
WEBDAV地址多了dav，在POTPLAYER里无法挂载

### DISCUSSION THREAD

**Discussion by fzlychee2019** [AlistGo/alist#1012] [Q&A] [answered]
往POTPLAYER添加WEBDAV地址时，因为地址多了/dav，变得端口号无法和地址匹配了，无法挂载。
![image](https://user-images.githubusercontent.com/61218953/165864741-daae031c-37ab-4045-ab36-c05305f14ff8.png)

**Comment by xhofe** [comment, marked as answer]
不用加`https://`协议头。
![image](https://user-images.githubusercontent.com/36558727/165894480-fd19c717-81d7-4b49-aa9a-1662590a797a.png)

**Reply by fzlychee2019** [reply]
非常感谢，去掉https，一切正常了，太棒了！

**Reply by sharewood8** [reply]
在alist界面，使用potplayer无法调用包含中文标题的文件，包括中文目录，改成全英文，就没有问题，是我哪里设置不对么？

**Reply by ssthptfpd** [reply]
为什么我挂载ALIST后，上面的阿里云盘跟天翼云盘的可以播放，上面的OD盘的文件就播放不了，提示地址错误。

**Reply by JSCSZZSS** [reply]
我去掉HTTPS也不对啊，还是报错无法修改文件夹

**Reply by gaowanliang** [reply]
> 我去掉HTTPS也不对啊，还是报错无法修改文件夹

默认密码不行，我换了个密码就好了

**Reply by liueic** [reply]
我也是 而且在手机端上的能连接服务器但是读不出来文件
